odoo/addons/base/models/qweb.py

```diff
@@ -35,18 +35,9 @@
 
 _logger = logging.getLogger(__name__)
 
-# in Python 2, arguments (within the ast.arguments structure) are expressions
-# (since they can be tuples), generally
-# ast.Name(id: identifyer, ctx=ast.Param()), whereas in Python 3 they are
-# ast.arg(arg: identifier, annotation: expr?) provide a toplevel arg()
-# function which matches ast.arg producing the relevant ast.Name in Python 2.
 arg = getattr(ast, 'arg', lambda arg, annotation: ast.Name(id=arg, ctx=ast.Param()))
-# also Python 3's arguments has grown *2* new mandatory arguments, kwonlyargs
-# and kw_defaults for keyword-only arguments and their default values (if any)
-# so add a shim for *that* based on the signature of Python 3 I guess?
 arguments = ast.arguments
-if pycompat.PY2:
-    arguments = lambda args, vararg, kwonlyargs, kw_defaults, kwarg, defaults: ast.arguments(args=args, vararg=vararg, kwarg=kwarg, defaults=defaults)
+
```

Comment:
These two helpers should be inlined since they only exist for P2/P3 compat: in P3, `arg` is an alias for `ast.arg`… and arguments is an alias for `ast.arguments`.